docs/core/testing/unit-testing-with-dotnet-test.md

```diff
@@ -8,101 +8,126 @@ ms.custom: "seodec18"
+* In the *unit-testing-using-dotnet-test* directory, run the following command to add the class library project to the solution:
 
-## Creating the test project
+  ```dotnetcli
+  dotnet sln add ./PrimeService/PrimeService.csproj
+  ```
 
-Next, create the *PrimeService.Tests* directory. The following outline shows the directory structure:
+* Create the *PrimeService.Tests* project by running the following command:
 
-```
-/unit-testing-using-dotnet-test
-    unit-testing-using-dotnet-test.sln
-    /PrimeService
-        Source Files
-        PrimeService.csproj
-    /PrimeService.Tests
-```
+  ```dotnetcli
+  dotnet new xunit -o PrimeService.Tests
+  ```
 
-Make the *PrimeService.Tests* directory the current directory and create a new project using [`dotnet new xunit`](../tools/dotnet-new.md). This command creates a test project that uses [xUnit](https://xunit.github.io/) as the test library. The generated template configures the test runner in the *PrimeServiceTests.csproj* file similar to the following code:
+* The preceding command:
+  * Creates the *PrimeService.Tests* project in the *PrimeService.Tests* directory. The test project uses [xUnit](https://xunit.github.io/) as the test library.
+  * Configures the test runner by adding the following `<PackageReference />`elements to the project file:
+     * "Microsoft.NET.Test.Sdk"
+     * "xunit"
+     * "xunit.runner.visualstudio"
 
-```xml
-<ItemGroup>
-  <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
-  <PackageReference Include="xunit" Version="2.2.0" />
-  <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
-</ItemGroup>
-```
+* Add the test project to the solution file by running the following command:
 
-The test project requires other packages to create and run unit tests. `dotnet new` in the previous step added xUnit and the xUnit runner. Now, add the `PrimeService` class library as another dependency to the project. Use the [`dotnet add reference`](../tools/dotnet-add-reference.md) command:
+  ```dotnetcli
+  dotnet sln add ./PrimeService.Tests/PrimeService.Tests.csproj
+  ```
 
-```dotnetcli
-dotnet add reference ../PrimeService/PrimeService.csproj
-```
+* Add the `PrimeService` class library as a dependency to the *PrimeService.Tests* project:
 
-You can see the entire file in the [samples repository](https://github.com/dotnet/samples/blob/master/core/getting-started/unit-testing-using-dotnet-test/PrimeService.Tests/PrimeService.Tests.csproj) on GitHub.
+  ```dotnetcli
+  dotnet add ./PrimeService.Tests/PrimeService.Tests.csproj reference ./PrimeService/PrimeService.csproj  
+  ```
 
-The following shows the final solution layout:
+<a name="create-test-cmd"></a>
 
-```
-/unit-testing-using-dotnet-test
-    unit-testing-using-dotnet-test.sln
-    /PrimeService
-        Source Files
-        PrimeService.csproj
-    /PrimeService.Tests
-        Test Source Files
-        PrimeServiceTests.csproj
-```
+### Commands to create test solution
```

Comment:
For me, "test" solution is confusing a little bit. The solution holds both projects, the test and the class lib.

Reply:
LOL, now you see why I added an anchor. Only one thing to change.

Reply:
Fixed. good suggestion.